contracts/seed/Seed.sol

```diff
@@ -127,132 +132,153 @@ contract Seed {
       * @param _tokens                Array containing two params:
                                         - The address of the seed token being distributed.
       *                                 - The address of the funding token being exchanged for seed token.
-      * @param _softAndHardCap         Array containing two params:
+      * @param _softHardReq           Array containing two params:
```

Comment:
Can we remove abbreviations like this?  So this could be "_softHardThresholds"

Reply:
Okay